services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/ios/controller/IosApiErrorHandler.java

```diff
@@ -82,6 +84,12 @@ public IosApiErrorHandler(SecurityLogger securityLogger) {
     return ResponseEntity.status(HttpStatus.TOO_MANY_REQUESTS)
         .body(of(errorCode));
   }
+  
+  @ExceptionHandler(ClientAbortException.class)
+  @ResponseStatus(HttpStatus.SERVICE_UNAVAILABLE)
```

Comment:
`@ResponseStatus(HttpStatus.SERVICE_UNAVAILABLE)` doesn't this then try to send something back?

Reply:
From all that I've seen, this is the correct way to handle it. Unfortunately, hard to test... :-/